src/directives/decorators/stb-webmanual/add-row-button.html

```diff
@@ -0,0 +1,6 @@
+<div class="form-group"  ng-show="showCondition()">
+    <button class="stb-btn transparent light"
+            type="button"
+            name="addRowButton"
+            ng-click="addRowClick($event,form)">{{ form.title }}</button>
```

Comment:
we should use ng-click="form.onClick()", so action (js function) must be defined in json of angular form instead of 'decorators.js'.